cncf/cncf.io
# New theme for CNCF

Getting to deploy:
1. Pull live db on to PR instance - _done, Apr 7_
2. Set up key main pages on PR instance for QA
3. QA period with internal team (Apr x-y)
4. Copy PR db and files on to DEV db and files
5. Merge PR (this will delete the PR instance)
6. UAT period with larger team (Apr y-z)

To deploy:
1. Freeze live instance (best to do this over a weekend)
2. Copy current dev instance on to `uat-dev` multidev instance
7. Copy live instance db/files on to `old-live` multidev instance
8. Copy live instance db/files on to the dev instance
9. Modify dev instance to make it ready to go live: copy key pages from `uat-dev` while checking for any updated content, theme switch, menus rebuild, site options, any other page updates, accessibility statement published, launch blog post published, report format updates
10. QA
11. Get approvals to push live
12. Copy dev db/files to live instance
13. Regenerate thumbnails `terminus wp cncfci.dev -- media regenerate --yes` and then make sure Shortpixel does its job
14. Check everything is working especially search/filter screens
15. Send unfreeze message and clean up
16. Title case all page titles so they show up consistently in the menu
17. Reconfigure Flying Scripts
